bool initialized_
float64 data